doc/user/bgp.rst

```diff
@@ -4010,6 +4010,11 @@ The following command is available in ``config`` mode as well as in the
    the startup configuration, graceful shutdown will remain in effect
    across restarts of *bgpd* and will need to be explicitly disabled.
 
+.. clicmd:: bgp inq-limit (0-4294967295)
+
+   Set the BGP In Queue limit for all peers when messaging parsing. Increase
```

Comment:
nit: can we use "Input Queue" instead of "In Queue"?

Reply:
sure, for both the command and doc, or just doc?

Reply:
Ideally both.  Would be nice to see in the help string too.

Reply:
done